app/workers/mail_chimp_worker.rb

```diff
@@ -0,0 +1,12 @@
+class MailChimpWorker
+  include Sidekiq::Worker
+  include Mailchimp
+
+  def perform(user_id)
+    user= User.find(@user.id)
```

Comment:
you meant `User.find(user_id)` ?

Reply:
Yeah, changed it.

Reply:
@Anagha22 could you push the changes that you have made?